layouts/partials/primary-download-matrix.hbs

```diff
@@ -64,12 +64,19 @@
       </tr>
 
       <tr>
-        <th>Linux Binaries (.tar.xz)</th>
+        <th>Linux Binaries (x86)</th>
```

Comment:
AFAIK `x86` is an instruction set and `x86-64` is an extension of `x86` so I'm also ok as is. I suggested the change because the links below are `linux-x86` and `linux-x64`.